src/resolver/tests/resolve_expressions_tests.rs

```diff
@@ -3869,35 +3877,35 @@ fn hardware_access_types_annotated() {
     if let AstNode { stmt: AstStatement::Assignment(Assignment { right, .. }), .. } =
         &unit.implementations[0].statements[0]
     {
-        assert_type_and_hint!(&annotations, &index, right, BYTE_TYPE, Some(BYTE_TYPE));
+        insta::assert_debug_snapshot!(annotations.get(right).unwrap());
```

Comment:
Any reason not to use inline-snapshots here?

Reply:
no reason, i can also do inline